python/rmm/_lib/device_buffer.pyx

```diff
@@ -139,3 +152,42 @@ cdef class DeviceBuffer:
 
     cdef void* c_data(self):
         return self.c_obj.get()[0].data()
+
+
+@cython.boundscheck(False)
+cpdef DeviceBuffer to_device(const unsigned char[::1] b, uintptr_t stream=0):
+    if b is None:
+        raise TypeError(
+            "Argument 'b' has incorrect type"
+            " (expected bytes, got NoneType)"
```

Comment:
I don't think this has to be bytes, IIRC Cython allows for this to be anything with buffer protocol, no?

Reply:
Yeah it should say `bytes`-like. Will update.

Reply:
This is corrected. Thanks for raising 🙂